src/main/java/org/example/homework_nr_5/HomeWork5.java

```diff
@@ -0,0 +1,82 @@
+package org.example.homework_nr_5;
+
+
+public class HomeWork5 {
+
+    public enum Weekday {
+        MONDAY ("weekday"),
```

Comment:
You can remove the "weekday" and "dayoff" strings near enums and in the methods below use something like:
`public boolean isWeekDay(){ if this != SUNDAY && this != SATURDAY) return true else return false;`

Reply:
i made changes in methods